nav2_docking/opennav_docking/src/docking_server.cpp

```diff
@@ -744,6 +774,15 @@ DockingServer::dynamicParametersCallback(std::vector<rclcpp::Parameter> paramete
       if (name == "max_retries") {
         max_retries_ = parameter.as_int();
       }
+    } else if(type == ParameterType::PARAMETER_BOOL){
+      if (name == "dock_backwards") {
+        dock_backwards_ = parameter.as_bool();
+      }
+      if (name == "backward_blind") {
+        backward_blind_ = parameter.as_bool();
+        initial_rotation_ = parameter.as_bool();
```

Comment:
You shouldn't update other parameters -- remove dock backward / initial rotation setting